server/protocols/webadmin-cli/src/test/java/org/apache/james/cli/DomainManageTest.java

```diff
@@ -58,4 +58,38 @@ void domainListCommandShouldWShowOnlyDefaultDomain(GuiceJamesServer server) {
         assertThat(outputStreamCaptor.toString().trim().toCharArray()).containsOnly("localhost".toCharArray());
     }
 
+    @Test
+    void domainCreateCommandWithValidNameShouldSuccessfully(GuiceJamesServer server) {
+        Port port = server.getProbe(WebAdminGuiceProbe.class).getWebAdminPort();
+
+        int exitCode = WebAdminCli.executeFluent(new PrintStream(outputStreamCaptor), new PrintStream(errorStreamCaptor),
+            "--url", "http://127.0.0.1:" + port.getValue(), "domain", "create", "linagora.com");
+        assertThat(exitCode).isEqualTo(0);
```

Comment:
Group all of the assertions at the end of your test please, it makes it more readable.

Reply:
Not done ?

Reply:
it's done